.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "files.eol": "\n",
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
What we should do with this? I think is a user pref, because I want it to be always on. Therefore, we should remove these settings as it will then be controlled by the global editor settings

Reply:
But anyways, good job.

Reply:
Yeah, good point. I removed it